README.md

```diff
@@ -2,41 +2,26 @@
 
 Generate simple markdown changelogs for GitHub repositories written in Python.
 
-This package does two things:
+This package provides a CLI to do two things:
 
-1. Given a GitHub org, repository, an initial git reference or date, use the
-   [GitHub GraphQL API](https://developer.github.com/v4/) to return a DataFrame
-   of all issue and PR activity for this time period.
-2. A CLI to render this activity as markdown, suitable for generating changelogs or
-   community updates.
+**Scrape all GitHub activity over a period of time for a repository**.
+Given a GitHub org, repository, an initial git reference or date, use the [GitHub GraphQL API](https://developer.github.com/v4/) to return a DataFrame of all issue and PR activity for this time period.
 
-_Note: This is a really young tool so it might change a bit over time._
+**Render this as a markdwon changelog**.
+Convert this DataFrame to markdown that is suitable for generating changelogs or   community updates.
 
-## Installation
+For an example, see the [changelog of this package]([https://](https://github-activity.readthedocs.io/en/latest/changelog)).
 
-The easiest way to install this package is via `pip`:
+## Use this tool
 
-```
-pip install github-activity
-```
-
-## Usage
-
-The easiest way to use `github-activity` to generate activity markdown is to use
-the command-line interface. It takes the following form:
+Use this tool via the command line like so:
```

Comment:
It wasn't clear to me whether this is intended as a quick start in which case this should probably mention an auth token is needed, or if it's an example of how easy this is to use in which case it might be better to say `After following the installation instructions you can use this tool via the command line like so:` ?